ops/services/app_gateway/main.tf

```diff
@@ -30,7 +30,7 @@ resource "azurerm_public_ip" "static_gateway" {
   location            = var.resource_group_location
   allocation_method   = "Static"
   sku                 = "Standard"
-  zones               = ["1", "2", "3"]
+  zones               = var.zones
```

Comment:
I see some using `var` and some using `local`. whats the difference?

Reply:
`var` references the variables being set up in the `_var.tf` files in the same directory, and because this Terraform code gets called via a module, we can pass in a value through that module block or use the default value set in the variable definition. 

`local` is a variable set in a `locals` block. In the case we have for `ops/prod/app_gateway_url_redirects.tf`, which is a prod-only file and not something we've set up as a reusable module, it makes sense to set it up via a `locals` block.